README.adoc

```diff
@@ -214,6 +214,7 @@ Services currently included:
 * `marlowe--sync`: `marlowe-sync` for the `preprod` network.
 * `marlowe--indexer`: `marlowe-indexer` for the `preprod` network.
```

Comment:
Change `--` to `-` in lines 214 and 215.

Reply:
I'll add this change to my next PR